.github/workflows/push_tests.yml

```diff
@@ -3,7 +3,7 @@ name: Slow Tests on main
 on:
   push:
     branches:
-      - main
+      - test-peft-unet
 
```

Comment:
Let's make sure to revert these changes :)

Reply:
Thanks for noticing!